### ISSUE TITLE
Does not work for JDK 12 + Clojure 1.10.0 

### ISSUE THREAD

**Issue by mjayprateek** [venantius/ultra#108]

**Comment by venantius** [comment]
Looks like an upstream issue in Fipp.

**Comment by xlxs4** [comment]
Exact same error here.

**Comment by zlrth** [comment]
>Looks like an upstream issue in Fipp.
Yes--an upstream issue in Fipp that's addressed by upgrading ultra's version of puget, right? I ran `lein deps :tree` on my project that uses ultra 0.6.0, and found, via puget 0.1.0, it was pulling an old version of fipp which is causing this runtime exception.

(i'm getting the same error)

**Comment by Ramblurr** [comment]
So is there a workaround?

**Comment by mrkaspa** [comment]
Having the same problem how can I try the fix?

**Comment by schmee** [comment]
Using `LEIN_USE_BOOTCLASSPATH=no lein repl` works for me. Source: https://github.com/brandonbloom/fipp/issues/60#issuecomment-470468952

**Comment by slikts** [comment]
`LEIN_USE_BOOTCLASSPATH=no` works but should at the very least be mentioned in the readme.

**Comment by slikts** [comment]
Applying the env workaround from the console was simple, but it was less simple to figure out how to get the VS Code remote extension to work; in the end, I had to add the variable to `.profile` and restart the remote server to be able to have a REPL in my editor.

**Comment by avysk** [comment]
It's almost four years old now. Any chance for a fix? I'm having the very same problem, Clojure 1.11.1, `Leiningen 2.10.0 on Java 17.0.6 OpenJDK 64-Bit Server VM`, `FreeBSD 13.1-RELEASE-p6 FreeBSD 13.1-RELEASE-p6 GENERIC amd64`.
